src/data/protocol-registry.ts

```diff
@@ -131,11 +131,20 @@ const protocols: WaylandProtocolRegistryItem[] = [
     },
     {
         id: 'input-method-unstable-v1',
-        name: 'Input method',
+        name: 'Input method v1',
         source: WaylandProtocolSource.WaylandProtocols,
         stability: WaylandProtocolStability.Unstable,
         protocol: require('./protocols/input-method-unstable-v1.json'),
     },
+    {
+        id: 'input-method-unstable-v2',
+        name: 'Input method v2',
+        source: WaylandProtocolSource.WlrProtocols,
+        stability: WaylandProtocolStability.Unstable,
+        protocol: require('./protocols/input-method-unstable-v2.json'),
+        externalUrl:
+            'https://gitlab.freedesktop.org/wlroots/wlroots/-/raw/dc3d1530/protocol/input-method-unstable-v2.xml',
```

Comment:
Minor nitpick to make this consistent with other GitLab external protocols (e.g.: `gtk-shell`):
```suggestion
            'https://gitlab.freedesktop.org/wlroots/wlroots/-/blob/dc3d1530/protocol/input-method-unstable-v2.xml',
```